### DISCUSSION TITLE
Query statistics

### DISCUSSION THREAD

**Discussion by marcuslindblom** [ravendb/ravendb#17776] [Q&A] [answered]
If I remember correctly `-1` means that the data is cached but I'm seeing a lot of `0` in my logs when using query statistics like this: 
```

const session = store.openSession();
const page = await session
  .query({ indexName: 'Content/ByUrl' })
  .whereEquals('url', Astro.url.pathname.replace(/\/$/, '')) // remove trailing slash
  .statistics(s => stats = s)
  .firstOrNull();
```
What does that mean?

**Comment by ayende** [comment, marked as answer]
That means that the query time was `<1ms`, basically.
We have a task to add more accurate timing data, which can capture requests that are that fast
